src/app/pages/HomePage/Graph/Graph/index.tsx

```diff
@@ -230,6 +246,16 @@ const LayerStatus: FC<{ scope: SearchScope; statusChange: (outOfDate?: boolean)
   return null
 }
 
+const LayerStatus: FC<{ scope: SearchScope; statusChange: (outOfDate?: boolean) => void }> = ({
+  scope,
+  statusChange,
+}) =>
+  scope.layer === Layer.consensus ? (
+    <ConsensusStatus network={scope.network} statusChange={statusChange} />
```

Comment:
1) On mobile when I click Consensus I don't see any Consensus status icon or tooltip. Is this is rendered currently?
2) When tooltip will be visible it will show wrong labels for Consensus (Open ParaTime -> Open,  '' -> 'ParaTime Online' something like that)

Reply:
>When tooltip will be visible it will show wrong labels for Consensus (Open ParaTime, ParaTime Online)

OK, fixed the tooltips.

Reply:
> On mobile when I click Consensus I don't see any Consensus status icon or tooltip. 

Me neither., but I think that is a separate issue, which we can solve in a separate PR.

The PR as it stands now already fixes the crash and the wrong working in the tooltips, so I consider it to worthy of merging, if there are no further questions regarding that part of the functionality.

Reply:
it's a part of the task "Validate if Consensus layer opens overlay with details" (tooltip)

Reply:
We discussed to do task 2 in a separate PR.